ci/provisioning/post_provision_config_nodes_EL_7.sh

```diff
@@ -48,34 +27,22 @@ timeout_yum() {
     return 1
 }
 
-disable_gpg_check() {
-    local url="$1"
-
-    repo="$(url_to_repo "$repo")"
-    # bug in EL7 DNF: this needs to be enabled before it can be disabled
-    dnf config-manager --save --setopt="$repo".gpgcheck=1
-    dnf config-manager --save --setopt="$repo".gpgcheck=0
-    # but even that seems to be not enough, so just brute-force it
-    if ! grep gpgcheck /etc/yum.repos.d/"$repo".repo; then
-        echo "gpgcheck=0" >> /etc/yum.repos.d/"$repo".repo
-    fi
+bootstrap_dnf() {
+    timeout_yum 5m install dnf 'dnf-command(config-manager)'
 }
 
-dump_repos() {
-        for file in /etc/yum.repos.d/*.repo; do
-            echo "---- $file ----"
-            cat "$file"
-        done
+group_repo_post() {
```

Comment:
(lint) The mentioned syntax error was in this function. [SC1009]
(lint) Couldn't parse this brace group. Fix to allow more checks. [SC1073]